tests/stp_test.py

```diff
@@ -0,0 +1,407 @@
+import os
+import pytest
+from click.testing import CliRunner
+
+import config.main as config
+import show.main as show
+from utilities_common.db import Db
+from .mock_tables import dbconnector
+
+
+show_spanning_tree = """\
+Spanning-tree Mode: PVST
+
+VLAN 500 - STP instance 0
+--------------------------------------------------------------------
+STP Bridge Parameters:
+Bridge           Bridge Bridge Bridge Hold  LastTopology Topology
+Identifier       MaxAge Hello  FwdDly Time  Change       Change
+hex              sec    sec    sec    sec   sec          cnt
+8064b86a97e24e9c 20     2      15     1     0            1
```

Comment:
I assume these comes from mock_tables. Consider using the json module and formatted string to generate the expected string. If this is too tedious might consider just asserting the full output string containing some substring like `0064b86a97e24e9c 600 806480a235f281ec Root 20 2 15`, which is usually good enough for catching bugs early.